### PR TITLE
Sample code for the article on basic types

### PR DESCRIPTION
**Pull request by lpozo**
**Where to put new files:**

- New files should go into a top-level subfolder, named after the article slug. For example: `my-awesome-article`

**How to merge your changes:** 

1. [Make sure the CI code style tests all pass (+ run the automatic code formatter if necessary).](https://github.com/realpython/materials/blob/master/README.md)
2. Find an RP Team member on Slack and ask them to review & approve your PR.
3. Once the PR has one positive ("approved") review, GitHub lets you merge the PR.
4. 🎉

### CODE REVIEW

File: python-basic-data-types/point.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    def __bool__(self):
+        if self.x == self.y == 0:
+            return False
+        return True
+
+
+origin = Point(0, 0)
+bool(origin)
+point = Point(2, 4)
+bool(point)
```
Comment (martin-martin) [inline python-basic-data-types/point.py:15]: I'd suggest to make this executable with some output, either like this:

```suggestion
origin = Point(0, 0)
print(bool(origin))
point = Point(2, 4)
print(bool(point))
```

Or, potentially wrap it into the name-main idiom, but maybe no need for that for this.

File: python-basic-data-types/escape_seqs.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+print("a\tb")
+print("a\nb")
+print("a\141\x61")
+print("\N{rightwards arrow}")
```
Comment (martin-martin) [inline python-basic-data-types/escape_seqs.py:4]: If you agree to add the octal and hex examples to the tutorial, please add them here as well.

**Review by martin-martin — CHANGES_REQUESTED** [review]
Two small things, thanks!

**Review by martin-martin — APPROVED** [review]
LGTM! 🎉

**Review by brendaweles — APPROVED** [review]

**Pull request merged**